201-vmss-master-slave-customscript/azuredeploy.json

```diff
@@ -0,0 +1,492 @@
+﻿{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmssName": {
+      "type": "String",
+      "defaultValue": "cloudmelonvmss",
+      "metadata": {
+        "description": "String used as a base for naming resources. Must be 3-10 characters in length and globally unique across Azure Stack. A hash is prepended to this string for some resources, and resource-specific information is appended."
```

Comment:
Azure Stack - VMSS is not supported on Azure Stack yet

Reply:
updated description.